gitlab/ci/webstandard.yml

```diff
@@ -1,8 +1,5 @@
 webstandard_check_role:
   image: domjudge/gitlabci:2.1
-  only:
```

Comment:
is this part intentional?

Reply:
No, the commit is marked as `[delete] ...` which means that before merging I would remove that commit. It makes that on gitlab we would also run the extra tests for admin as that's why I broke `main`, we only test a subset of the pages in the `webstandard` job to speed up the CI, when I set it up that way I accepted the small risk that we would break `main` as what happened now.

When we move to `GHA` we can test everything as they make it possible to stop matrix jobs when one of them fails.